### PR TITLE
Add category and expense command tests

### LINKED ISSUE #74: Add Testing
Add test cases for all the new methods to pass CodeCov tests.

### PR DESCRIPTION
**Pull request by shirsho-12**
Closes #74

### CODE REVIEW

File: src/test/java/seedu/address/storage/JsonExpenseTrackerStorageTest.java
Comment (nicleejy) [inline src/test/java/seedu/address/storage/JsonExpenseTrackerStorageTest.java:1]: Overall, the code looks good to me.

File: src/test/java/seedu/address/logic/commands/AddExpenseCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    @Test
+    public void addExpenseCommandTest() throws CommandException {
+        requireNonNull(model);
+        model.addExpense(expense);
+        model.addExpense(expense2);
+        model.addExpense(expense3);
+        assertEquals(model.getExpenses().get(0), expense);
+        assertEquals(model.getExpenses().get(1), expense2);
+        assertEquals(model.getExpenses().get(2), expense3);
```
Comment (nicleejy) [inline src/test/java/seedu/address/logic/commands/AddExpenseCommandTest.java:33]: LGTM! Thanks for the fix.

**Review by nicleejy — APPROVED** [review]

File: src/test/java/seedu/address/commons/stubs/ModelStub.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    @Override
+    public void setExpenseTracker(ReadOnlyExpenseTracker expenseTracker) {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'setExpenseTracker'");
+    }
+
+    @Override
+    public ReadOnlyExpenseTracker getExpenseTracker() {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'getExpenseTracker'");
```
Comment (gitsac) [inline src/test/java/seedu/address/commons/stubs/ModelStub.java:119]: Great thinking to use this as a placeholder for un-done tests!

File: src/main/java/seedu/address/logic/commands/AddCategory.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import seedu.address.model.category.Category;
 
 /**
- * Adds a category to the address book.
+ * Adds a category to the Expense Tracker.
```
Comment (gitsac) [inline src/main/java/seedu/address/logic/commands/AddCategory.java:12]: Good to remove traces of AB3 and replace it with our group's own take on it.

**Review by gitsac — APPROVED** [review]
LGTM.

**Pull request merged**